pkg/proxy/iptables/proxier.go

```diff
@@ -1283,6 +1283,15 @@ func (proxier *Proxier) syncProxyRules() {
 				} else if svcInfo.Protocol() != v1.ProtocolSCTP {
 					socket, err := proxier.portMapper.OpenLocalPort(&lp)
 					if err != nil {
+						msg := fmt.Sprintf("can't open %s, skipping this nodePort: %v", lp.String(), err)
```

Comment:
How about "can't open NodePort %s, skipping it: %v" - simpler

Reply:
Ok. I fixed it by additional commit to fix similar other messages.